.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build_and_test:
+    macos:
+      xcode: "10.1.0"
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - wordpress-authenticator-ios-{{ checksum "Gemfile.lock" }}-{{ checksum "Podfile.lock" }}
+      - run:
+          name: Bundle install
+          command: bundle install --path=vendor/bundle
+      - run:
```

Comment:
CircleCI recommend using this step prior to running `pod install` (notice that `pod install` takes 8 minutes currently):

```
- run:
          name: Fetch CocoaPods Specs
          command: |
            curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
```

As we expand more across CircleCI I think it may may sense to share steps using [CircleCI Orbs](https://circleci.com/orbs/) so we don't need to duplicate things like this.